### DISCUSSION TITLE
How to display scrollbar in ActionMenu?

### DISCUSSION THREAD

**Discussion by callmekatootie** [primer/react#3193] [Q&A]
I am following the [minimal example](https://primer.style/react/ActionMenu#minimal-example) to create an ActionMenu. I have too many `<ActionList.Item/>` components.

When I click on the button, the overlay does appear - but there's no scrollbar so I can only see part of the content. I checked for props that I could enable but could not find one. Should I manually display the scrollbar through CSS?

![image](https://user-images.githubusercontent.com/2317133/233162540-b8c5e805-82b9-47e7-9be9-e041befa346a.png)

**Comment by lesliecdubs** [comment]
Hi, thanks for the question!

@langermank (First Responder) do you have any guidance to share around best practices for enabling scrolling within a long `ActionMenu`?

**Comment by langermank** [comment]
@callmekatootie I believe you can pass in a `height` which will enable a scrollable overlay. It seems like `auto` height is the default. Here's the [source code](https://github.com/primer/react/blob/main/src/Overlay/Overlay.tsx#L16).

**Reply by callmekatootie** [reply]
That does set the height of the overlay, but there's no scrollbar. The overflowing content still gets hidden.

**Comment by cbix** [comment]
I always set the overflow behavior manually:
```tsx
<ActionMenu>
  <ActionMenu.Overlay width="auto" height="small" sx={{ overflowY: 'auto' }}>
    <ActionList>
      {/* items */}
    </ActionList>
  </ActionMenu.Overlay>
</ActionMenu>
```
![image](https://user-images.githubusercontent.com/1295945/236203092-d6d6fb0a-c332-472d-8366-d961de1c84e2.png)
However I couldn't find a way yet to automatically scroll to a specific list item, tried different things (`active`, `initialFocusRef`, `autoFocus`, even calling `scrollIntoView` on the overlay opening callback) but none of them worked ...
